src/components/UniversalNavbarExpanded/DropdownNav/DropdownChildren.js

```diff
@@ -0,0 +1,105 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import get from 'lodash/get'
+
+// Reused assets from UniversalNavbar
+import { DropdownLinkIcon } from '../../UniversalNavbar/assets/icons.js'
+
+// EDS core components
+import { Layout, Footnote } from '../../index'
+
+// Parent component (UniversalNavbar) siblings
+import NavLink from '../NavLink'
+
+// DropdownChildren sibling components
+import DropdownCta from './DropdownCta'
+
+// Helpers
+import { removeLastWord, getLastWord } from '../../../helpers/splitLastWord'
+
+// Styles
+import styles from './DropdownChildren.module.scss'
+
+/**
+ * Content (links) displayed in the dropdown menu after hovering the parent element
+ *
+ * @param {string} extraClass - Extra top level class
+ * @param {object} child - CTA copy, url and link items with titles/urls
+ * @param {object} LinkComponent - Agnotistic Reach and React Router Link (ex. Gatsby's <Link>)
+ *
+ * @return {JSX.Element}
+ */
+const DropdownChildren = ({ extraClass, child, LinkComponent }) => {
+  // Split the links for the subnav into two columns
```

Comment:
I'm not sure I follow what's going on here exactly

Reply:
It has to do with the way design wanted spacing between each column of links in the sub-navigation to be spaced. The order of the links should be:
```
one    ||   two
three ||   four
five    ||   six
```
So I'm breaking the original array of links into two separate columns using the modulus condition to alternate pushing between the first and second column. 

I'll add some more descriptive commenting to the file.

Reply:
8b0d3cf